build.sc

```diff
@@ -419,6 +420,14 @@ object main extends MillModule {
       "-DMILL_GRAPHVIZ=" + runClasspath().map(_.path).mkString(",")
     )
   }
+
+  object testkit extends MillModule {
```

Comment:
`MillModule` is inheriting `MillApiModule` which also adds `MiMa`. Instead, you could inherit from `MillInternalModule` directly. As a consequence, you need to add the `object test` manually or also inherit `MillAutoTestSetup`.